controllers/openstackclient_controller.go

```diff
@@ -209,6 +209,16 @@ func (r *OpenStackClientReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 	tripleoDeployFiles := tripleoDeployCM.Data
 	templateParameters["TripleoDeployFiles"] = tripleoDeployFiles
 
+	osbmsList := &ospdirectorv1beta1.OpenStackBaremetalSetList{}
+	listOpts := []client.ListOption{
+		client.InNamespace(instance.Namespace),
+	}
+
+	err = r.Client.List(context.TODO(), osbmsList, listOpts...)
+	if err != nil {
+		return ctrl.Result{}, err
+	}
+
```

Comment:
Are we actually using osbmsList anywhere?

Reply:
no, no longer, removed in latest update